src/bindings/python/pycsp.c

```diff
@@ -865,6 +865,11 @@ static PyObject* pycsp_kiss_init(PyObject *self, PyObject *args) {
 static PyObject* pycsp_sband_init(PyObject *self, PyObject *args) {
     csp_iface_t *sband_iface = 0;
     sdr_conf_t sdr_conf = {0};
+    bool using_fec;
+    if (!PyArg_ParseTuple(args, "b", &using_fec)) {
```

Comment:
This was probably in the ground station README.md, but how does this actually look on the command line? Just a bit surprised to see "b" == use FEC, since "b" is often reserved for baud rate.

Reply:
This isn't argument parsing from the command line. This looks at the values passed to the function. The string specifies the types PyArg_ParseTuple will use

Reply:
Bleh. I remember now. I guess I should be thankful for the C to Python binding, it really is an enormous amount of work.